pkg/cosign/verify.go

```diff
@@ -281,6 +337,11 @@ func VerifyImageSignatures(ctx context.Context, signedImgRef name.Reference, co
 	return checkedSignatures, bundleVerified, nil
 }
 
+func isb64(data []byte) bool {
```

Comment:
A suggestion. Why not inline this in the main function? This reduces the readability from jumping from one function to another. In go it is generally not a practice to have a small function with a simple check.

Reply:
I agree with @naveensrinivasan, wrapping a function call inside of another function isn't ideal.

Reply:
thank you @hectorj2f, done.